task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +205,11 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+    if ( Math.sqrt((circle.center.x - point.x)**2 + (circle.center.y - point.y)**2) - circle.radius < 0) {
+        return true
+    }
+    else 
+        return false
```

Comment:
We can simplify this logic by using just one 'return'
Please, see https://javascript.info/logical-operators